examples/liveness/README.md

```diff
@@ -0,0 +1,75 @@
+## Overview
+This example shows two types of pod health checks: HTTP checks and containter execution checks.
+
+The **exec-liveness.yaml** demonstrates the container execution check.
+```
+    livenessProbe:
+      exec:
+        command:
+        - cat
+        - /tmp/health
+      initialDelaySeconds: 15
+      timeoutSeconds: 1
+```
+The container executes the command 'cat /tmp/health'. The health check fails if the command has an exit code that is not 0.
+
+Note that the docker image's cmd in exec-liveness.yaml remove the '/tmp/health' after 10 seconds,
```

Comment:
Note that the container removes the `/tmp/health` file after 10 seconds, so when Kubelet executes the health check 15 seconds (defined by *initialDelaySeconds*) after the container started, the check would fail.